services/app-web/src/hooks/useContractForm.ts

```diff
@@ -0,0 +1,260 @@
+        setShowPageErrorMessage(false)
+        const { populationCovered, programIDs, riskBasedContract, submissionType, submissionDescription, contractType } = input
+        try {
+            const createResult = await createFormData({
+                variables: {
+                    input: {
+                        populationCovered,
+                        programIDs,
+                        riskBasedContract,
+                        submissionType,
+                        submissionDescription,
+                        contractType,
+                },
+            }})
+            const createdSubmission: Contract | undefined =
+                createResult?.data?.createContract.contract
+
+            if (!createdSubmission) {
+                setShowPageErrorMessage(true)
+                console.info('Failed to update form data', createResult)
+                recordJSException(
+                    `StateSubmissionForm: Apollo error reported. Error message: Failed to create form data ${createResult}`
+                )
+                return new Error('Failed to create form data')
+            }
+
+            return createdSubmission
+        } catch (serverError) {
+            setShowPageErrorMessage(true)
+            recordJSException(
+                `StateSubmissionForm: Apollo error reported. Error message: ${serverError.message}`
+            )
+            return new Error(serverError)
+        }
+    }
+    const [updateFormData] = useUpdateContractDraftRevisionMutation()
+
+    const updateDraft: UseContractForm['updateDraft']  = async (
+        input: UpdateContractDraftRevisionInput
+    ): Promise<Contract | Error> => {
+
+        setShowPageErrorMessage(false)
+        if (input.formData.contractDocuments && input.formData.contractDocuments.length > 0) {
+            input.formData.contractDocuments = documentsInput(input.formData.contractDocuments)
+        } if (input.formData.supportingDocuments && input.formData.supportingDocuments.length > 0) {
+            input.formData.supportingDocuments = documentsInput(input.formData.supportingDocuments)
+        } if (input.formData.stateContacts && input.formData.stateContacts.length > 0) {
+            input.formData.stateContacts = stateContactsInput(input.formData.stateContacts)
+        }
+
+        try {
+            const updateResult = await updateFormData({
+                variables: {
+                    input: {
+                        contractID: contractID ?? 'new-draft',
+                        lastSeenUpdatedAt: contract!.draftRevision!.updatedAt,
+                        formData: input.formData
+                    },
+                },
+            })
+            const updatedSubmission =
+                updateResult?.data?.updateContractDraftRevision.contract
+            if (!updatedSubmission) {
+                setShowPageErrorMessage(true)
+                console.info('Failed to update form data', updateResult)
```

Comment:
FYI everywhere with `console.info` preceeding `recordJSException` we can delete the inline console. The record exception function also does a `console.error` now.